library/init/meta/interactive.lean

```diff
@@ -1559,6 +1559,10 @@ private meta def add_interactive_aux (new_namespace : name) : list name → comm
   (name.mk_string h _) ← return d_name,
   let new_name := `tactic.interactive <.> h,
```

Comment:
This function does not use the `new_namespace` argument anywhere. Instead, it hard codes `tactic.interactive`. Can we use the argument instead?

Reply:
good catch! I fixed this and added a test.